src/core/helper/tests/test_cache.py

```diff
@@ -0,0 +1,3 @@
+def test_strReplace():
+    string = "Hello, World!"
+    assert string.replace("H", "J") == "Jello, World!"
```

Comment:
Can this be removed?

Reply:
sure, i all add some real tests there XD